dts/bindings/misc/skyworks,sky13317.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2022 Stancu Florin
+# SPDX-License-Identifier: Apache-2.0
+
+description: Skyworks SKY13317 pHEMT GaAs SP3T Antenna Switch
+
+compatible: "skyworks,sky13317"
+
+include: [pinctrl-device.yaml, base.yaml]
+
+properties:
+  v1-gpios:
+    type: phandle-array
+    required: true
+    description: v1 control pin
```

Comment:
For me, the 'vX' description isn't very clear. Why 'v'?

I will say, are we overcomplicating this by taking the perspective of the antenna switch rather than the perspective of integrating the SoC's wireless framework and *possibly* needing to set GPIOs based on various modes?